chart/openfaas/values.yaml

```diff
@@ -15,13 +15,19 @@ gateway:
   readTimeout : "20s"
   writeTimeout : "20s"
   upstreamTimeout : "15s"  # Must be smaller than read/write_timeout
+  replicas: 1
 
 queueWorker:
   ackWait : "30s"
+  replicas: 1
 
 # image pull policy for openfaas components, can change to `IfNotPresent` in offline env
 openfaasImagePullPolicy: "Always"
 
+# replaces faas-netes with openfaas-operator
+operator:
+  create: true
```

Comment:
Should not be default as it is likely to have unexpected side effects, should be false with update in readme explaining how to switch.